eng/config/PublishData.json

```diff
@@ -60,258 +60,43 @@
       "vsMajorVersion": 16
     },
-    "release/dev16.10-vs-deps": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "3.10.*",
-      "packageFeeds": "default",
-      "channels": [],
-      "vsBranch": "rel/d16.10",
-      "vsMajorVersion": 16,
-      "insertionTitlePrefix": "[d16.10]"
-    },
     "release/dev16.11-vs-deps": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "3.11.*",
-      "packageFeeds": "default",
-      "channels": [],
       "vsBranch": "rel/d16.11",
       "vsMajorVersion": 16,
       "insertionTitlePrefix": "[d16.11]"
     },
     "release/dev17.0-vs-deps": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "4.0.*",
-      "packageFeeds": "default",
-      "channels": [],
       "vsBranch": "rel/d17.0",
       "vsMajorVersion": 17,
       "insertionTitlePrefix": "[d17.0]"
     },
     "release/dev17.1": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "4.1.*",
-      "packageFeeds": "default",
-      "channels": [],
       "vsBranch": "rel/d17.1",
       "vsMajorVersion": 17,
       "insertionTitlePrefix": "[d17.1]"
     },
     "release/dev17.2": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "4.2.*",
-      "packageFeeds": "default",
-      "channels": [],
       "vsBranch": "rel/d17.2",
       "vsMajorVersion": 17,
       "insertionTitlePrefix": "[d17.2]"
     },
     "main": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "4.3.*",
-      "packageFeeds": "default",
-      "channels": [],
       "vsBranch": "main",
       "vsMajorVersion": 17,
       "insertionCreateDraftPR": false,
       "insertionTitlePrefix": "[d17.3p1]"
-    },
-    "features/NullableReferenceTypes": {
-      "nugetKind": "PerBuildPreRelease",
-      "version": "2.6.*",
-      "nuget": [],
-      "vsix": null,
-      "channels": [
-        "nonnull"
-      ],
-      "vsBranch": "lab/d16.1stg",
-      "vsMajorVersion": 16
-    },
-    "features/dataflow": {
-      "nugetKind": "PerBuildPreRelease",
-      "version": "2.8.*",
-      "nuget": [],
-      "vsix": null,
-      "channels": [
-        "dataflow"
-      ],
-      "vsBranch": "lab/d16.1stg",
-      "vsMajorVersion": 16
-    },
-    "features/razorSupport2": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "3.1.*",
-      "nuget": [],
-      "vsix": null,
-      "channels": [],
-      "vsBranch": "lab/d16.1stg",
-      "vsMajorVersion": 16
-    },
-    "features/compilerNext": {
-      "nugetKind": [
-        "Shipping",
-        "NonShipping"
-      ],
-      "version": "3.4.*",
-      "nuget": [],
-      "vsix": null,
-      "channels": [],
-      "vsBranch": "main",
-      "vsMajorVersion": 16
-    },
-    "features/unitTesting": {
-      "nugetKind": [
```

Comment:
it might be good to document (somewhere) that these are available fields, now that there aren't any examples

For example it looks like we've used different nugetkinds and channels in the past (e.g. nullable ref types set channel and nugetkind)

Reply:
Good call, looks like nugetKind is actually used in publishing and I didn't realize it.